src/calculator.js

```diff
@@ -0,0 +1,32 @@
+import './calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
```

Comment:
- [ ] I kindly suggest that you follow the given design colors for the Calculator.
- Use a more similar orange color for the `/`, `x`, `-`, `+`, `-` button.
- Remove the space between the top and bottom part.

| Your Design  | Given Design |
| ------------- | ------------- |
|<img width="330" alt="Screenshot 2023-03-05 005423" src="https://user-images.githubusercontent.com/52934128/222934305-896f461d-3865-42cf-886d-ef1c7fb63569.png">  | ![image](https://user-images.githubusercontent.com/52934128/222933726-e8083830-0acc-42d7-b46c-5ae983c3328d.png)|